pkg/reconciler/cluster/controller.go

```diff
@@ -11,10 +11,11 @@ import (
 	clusterv1alpha1 "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/typed/cluster/v1alpha1"
 	"github.com/kcp-dev/kcp/pkg/client/informers/externalversions"
 	"github.com/kcp-dev/kcp/pkg/syncer"
+	"github.com/kcp-dev/kcp/pkg/util/errors"
```

Comment:
Is this file included in this PR?

In my experience `util` packages tend to grow and eventually include all kinds of gross stuff. Can this just be `pkg/errors`?

Reply:
I'm not sure it's really worth adding a package at the top-level just for such a small file: https://github.com/kcp-dev/kcp/pull/90/files#diff-0ec7ddef9b8a85c8f11ad3d3404465abd9494ec09d326dbb5377691e9a69aa18
I understand your concern though, and I assume we could call the package `common/errors` if `util` is a problem for you. I mainly copied the `util` pattern on what is being done at several places in the `kubernetes` repo itself.

Reply:
What about keeping it like this for now (no different from what K8S does), merge this PR, and open an issue to discuss what should be the preferred way to handle common utilities in the future ?
When there is an agreement on the issue, we would be able to just move the `errors.go` where it will belong.
@smarterclayton any strong opinion here ?

Reply:
I'm fine merging this as-is, but we should stay vigilant about our hygiene in this package. Whether it's called `common` or `util`, these catch-all shared packages tend to become big messy dependency bombs in my experience.